src/context/useLocalPeerings.tsx

```diff
@@ -0,0 +1,21 @@
+import { queryKeys } from "util/queryKeys";
+import type { UseQueryResult } from "@tanstack/react-query";
+import { useQuery } from "@tanstack/react-query";
+import type { LxdNetwork, LxdNetworkPeer } from "types/network";
+import { fetchNetworkPeers } from "api/network-local-peering";
+
+export const useLocalPeerings = (
```

Comment:
Good idea to introduce this, also I think for this layer it is ok to use the UI wording as is.